bridge/slack/slack.go

```diff
@@ -195,16 +195,16 @@ func (b *Bslack) Send(msg config.Message) (string, error) {
 
 	// Use webhook to send the message
 	if b.GetString(outgoingWebhookConfig) != "" {
-		return b.sendWebhook(msg)
+		return "", b.sendWebhook(msg)
 	}
 	return b.sendRTM(msg)
 }
 
 // sendWebhook uses the configured WebhookURL to send the message
-func (b *Bslack) sendWebhook(msg config.Message) (string, error) {
```

Comment:
Removing the returned `string` as suggested by the `unparam` linter. It's value is always `""`.